subsys/bluetooth/controller/ll_sw/ull_sync_types.h

```diff
@@ -33,6 +40,8 @@ struct ll_sync_set {
 		};
 	} node_rx_lost;
 
+	struct node_rx_hdr *node_rx_sync_estab;
+
```

Comment:
Unused variable?

Reply:
It is used but in commits following this commit e.g. https://github.com/zephyrproject-rtos/zephyr/blob/3c3805142656758a24fdf853cbcc9f8fed3159ce/subsys/bluetooth/controller/ll_sw/ull_sync.c#L158